content/webapp/pages/homepage.js

```diff
@@ -96,28 +96,38 @@ export class HomePage extends Component<Props> {
             </PageHeading>
             <TogglesContext.Consumer>
               {({ buildingReopening }) =>
-                buildingReopening ? (
-                  <Space
-                    v={{
-                      size: 'm',
-                      properties: ['padding-top', 'padding-bottom'],
-                    }}
-                  >
-                    <ReopeningBanner />
-                  </Space>
-                ) : (
+                !buildingReopening &&
+                standFirst && (
                   <>
```

Comment:
do you need this?

Reply:
I _think_ so. If you don't have the reopening toggle on, then you should still see the beige box that's currently there (as long as that content exists).